Makefile

```diff
@@ -191,6 +191,8 @@ coverage-test: coverage-build
 	$(RM) out/$(BUILDTYPE)/obj.target/node/gen/*.gcda
 	$(RM) out/$(BUILDTYPE)/obj.target/node/src/*.gcda
 	$(RM) out/$(BUILDTYPE)/obj.target/node/src/tracing/*.gcda
+	$(RM) out/$(BUILDTYPE)/obj.target/node_lib/{src,gen}/*.gcda
```

Comment:
Can you write this as two rules so it would be accepted by GNU make 4.1? See https://github.com/nodejs/node/pull/18332

Reply:
done! thanks.